projects/gfbio-app/src/app/app.component.html

```diff
@@ -2,10 +2,16 @@
     <mat-sidenav position="end" mode="side" (openedChange)="mapComponent.resize()">
         <wave-sidenav-container></wave-sidenav-container>
     </mat-sidenav>
-    <div class="time-container" fxLayoutAlign="space-around center">
+    <div class="time-container" fxLayoutAlign="space-between start">
+        <a href="https://www.gfbio.org" mat-flat-button color="default" class="back-button mat-elevation-z4">
```

Comment:
well i think we can do this but i'm not convinced it is the correct location. however, lets keep it for now.